<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Resultado</class>
 <widget class="QDialog" name="Resultado">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>302</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Execução Resultado</string>
  </property>
  <property name="locale">
   <locale language="Portuguese" country="Brazil"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Número de iterações:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelIteracoes">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Quantidade de passos até o objetivo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelQtdPassos">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="passoAnterior">
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playPause">
       <property name="text">
        <string>|&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="proximoPasso">
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Passo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPasso">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tablePuzzle">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>3</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string>0</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelObjetivo">
     <property name="text">
      <string>Objetivo!</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok">
       <property name="text">
        <string>Ok!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
